client/rest-high-level/build.gradle

```diff
@@ -92,7 +92,6 @@ check.dependsOn(asyncIntegTest)
 testClusters.all {
   testDistribution = 'OSS'
   systemProperty 'opensearch.scripting.update.ctx_in_params', 'false'
-  setting 'reindex.remote.whitelist', '[ "[::1]:*", "127.0.0.1:*" ]'
```

Comment:
Can we instead add regex which covers the new patterns we want to match?

Reply:
The pattern that we need to match is <opensearch endpoint>:<port> . But this is already covered by the wildcards that exist.
So for both IP address and port, (*) wildcard can be specified which makes it equal to removing the whitelist.
So it seems, we can leave the code as it and leave to the user to decide whether to have an explicit whitelist or use wildcard to bypass the whitelist.